requirements/deploy.txt

```diff
@@ -115,7 +115,7 @@ django==3.2.16
     #   django-taggit
     #   djangorestframework
     #   jsonfield
-django-allauth==0.51.0
+django-allauth==0.52.0
```

Comment:
It introduces 2 things we may want to check:

> * Introduced a new provider setting OAUTH_PKCE_ENABLED that enables the PKCE-enhanced Authorization Code Flow for OAuth 2.0 providers.
> * When ACCOUNT_PREVENT_ENUMERATION is turned on, enumeration is now also prevented during signup, provided you are using mandatory email verification. There is a new email template (templates/account/email/acccount_already_exists_message.txt) that will be used in this scenario.


https://django-allauth.readthedocs.io/en/latest/release-notes.html#note-worthy-changes

Reply:
It seems the new changes don't affect us. So, we are good to upgrade here 👍🏼

Reply:
We need to create the html template for `templates/account/email/acccount_already_exists_message.html`